internal/output/progress_unix.go

```diff
@@ -0,0 +1,8 @@
+//go:build linux || darwin
+// +build linux darwin
+
+package output
+
+func (d *Spinner) moveCaretBackInCommandPrompt(n int) {
+	// No-op
```

Comment:
Let's send a rollbar error if this happens, since it should never happen.

Reply:
I thought about that, but then we'll hammer rollbar on every tick.

Reply:
Could just only send it the first time. Record it to something like `Spinner.recorded = true`.

Reply:
Aye, but is there a chance we'll be creating a large number of these progress objects? We'd still be in the same boat.